typing/build.gradle.kts

```diff
@@ -42,7 +42,7 @@ dependencies {
     implementation(KoinDependencies.koinViewModel)
 
     //to get dynamic feature module
-    implementation(PlayCoreDependencies.playCore)
+    implementation(GooglePlayDependencies.playCore)
```

Comment:
why do we need GooglePlayDependencies instead of PlayCoreDependencies

Reply:
I just renamed this class cause now we have two Google Play dependencies instead of just play core. Let me know if you have suggestion for a better name for this use case